### DISCUSSION TITLE
The UI is sluggish and freezes

### DISCUSSION THREAD

**Discussion by xeRicker** [GitbookIO/community#484] [0. General bugs and questions]
Since the new UI changes, the site's been very laggy for me. It’s freezing constantly when editing text for about 4-5 seconds, and refreshing the page takes ages.

**Comment by addisonschultz** [comment]
Hey @xeRicker - Sorry to hear you're having some issues with the updated UI - it should definitely be faster! 

Do you have any additional information you can provide on the space you're working in? Feel free to send it over to `support@gitbook.com` so we can investigate this a bit 👍

**Comment by benzitney** [comment]
I have had the same experience and have sent in some support tickets. This most commonly manifests itself for me when trying to drag elements to different sections.
